web-client/src/components/MainLayout/index.tsx

```diff
@@ -1,118 +1,143 @@
+                    <NavLinkLogo to="/"><img src={Logo} alt="Logo" /></NavLinkLogo>
+                    <Box sx={{ width: '100px', height: '40px' }}>
+                        <FormControl fullWidth>
+                            <InputLabel id="demo-simple-select-label">Language</InputLabel>
+                            <Select
+                                labelId="demo-simple-select-label"
+                                id="demo-simple-select"
+                                style={{ height: '35px' }}
+                                value={lang}
+                                label="Language"
+                                onChange={handleChange}
+                            >
+                                <MenuItem value="1">En</MenuItem>
+                                <MenuItem value="2">UA</MenuItem>
+                            </Select>
+                        </FormControl>
                     </Box>
-                </Toolbar>
-            </AppBar>
-            <Box component="nav">
-                <Drawer
-                    container={container}
-                    variant="temporary"
-                    open={mobileOpen}
-                    onClose={handleDrawerToggle}
-                    ModalProps={{
-                        keepMounted: true, // Better open performance on mobile.
-                    }}
+                </FlexBox>
+                <FlexBox direction="column" padding="16px 8px 8px 8px">
+                    {children}
+                </FlexBox>
+                <BottomNavigation
                     sx={{
-                        display: { xs: 'block', sm: 'none' },
-                        '& .MuiDrawer-paper': { boxSizing: 'border-box', width: drawerWidth },
-                    }}
-                >
-                    {drawer}
-                </Drawer>
-            </Box>
-            <Box component="main" sx={{ p: 3 }}>
-                <Toolbar />
-                {children}
-            </Box>
-        </Box>
-    )
+                        boxShadow: '0px 3px 6px rgba(0, 0, 0, 0.1)',
+                        // borderRadius: '15px'
+                    }}>
+                    <FlexBox justify="space-between">
+                        {appRoutesMap.map((route) => (
+                            <NavLink to={route.to} key={route.name}>
+                                <BottomNavigationAction
+                                    label={route.name}
+                                    icon={route.icon}
+                                    sx={{
+                                        color: pathname === route.to ? '#ff5050' : 'grey',
+                                    }}
+                                />
+                            </NavLink>
+                        ))}
+                    </FlexBox>
+                </BottomNavigation>
+            </Paper>
+        ) : (
+            <FlexBox backgroundColor="white">
+                <CssBaseline />
```

Comment:
we don't need the CssBaseline in isMobile mode?

Reply:
I moved it to the root of the project to provide basic styles and style normalization for the entire app